rootston/xwayland.c

```diff
@@ -35,15 +53,20 @@ void handle_xwayland_surface(struct wl_listener *listener, void *data) {
 	wl_list_init(&roots_surface->destroy.link);
 	roots_surface->destroy.notify = handle_destroy;
 	wl_signal_add(&surface->events.destroy, &roots_surface->destroy);
+	wl_list_init(&roots_surface->request_configure.link);
+	roots_surface->request_configure.notify = handle_configure;
+	wl_signal_add(&surface->events.request_configure,
+		&roots_surface->request_configure);
 
 	struct roots_view *view = calloc(1, sizeof(struct roots_view));
 	view->type = ROOTS_XWAYLAND_VIEW;
-	view->x = view->y = 200;
+	view->x = (double) surface->x;
+	view->y = (double) surface->y;
```

Comment:
Here as well

Reply:
Can you also open a ticket with a list of x11 clients you've tested? We'll also fill it with clients we *should* test and try to deal with any weird ones.